src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write code here
+const wallImage = document.querySelector('div.wall');
+
+wallImage.style.display = 'flex';
+wallImage.style.alignItems = 'center';
+wallImage.style.justifyContent = 'center';
+wallImage.children[0].style.position = 'unset';
+
+// document.querySelector('img').style.top = '175px';
+// document.querySelector('img').style.left = '175px';
+// extra short variant;
```

Comment:
Good job! But now let's reimplement the task using element properties from the lesson

Reply:
ok)

Reply:
Done